contracts/contracts/vault/VaultCore.sol

```diff
@@ -565,6 +564,27 @@ contract VaultCore is VaultInitializer {
                     Pricing
     ****************************************/
 
+    /**
+     * @notice The amount of ETH value for redeeming 1 OETH.
+     * This is the minimum price for OETH. The better price is
+     * is usually achieved by swapping OETH for ETH
+     * with the Curve OETH/ETH pool.
+     */
+    function floorPrice() external view returns (uint256 price) {
+        // Get the assets for redeeming 1 OETH
+        // This has already had the redeem fee applied
+        uint256[] memory redeemAssets = _calculateRedeemOutputs(1e18);
+
+        // For each of the redeemed assets
+        for (uint256 i = 0; i < redeemAssets.length; ++i) {
+            // Sum the value of the asset in ETH = asset amount * oracle price
+            price +=
+                redeemAssets[i] *
+                IOracle(priceProvider).price(allAssets[i]);
+        }
+        price = price / 1e18;
```

Comment:
Makes a lot of sense to do the division once. Maybe a comment in the code on it?

Reply:
good idea. Done